devito/operator.py

```diff
@@ -193,6 +197,15 @@ def signature(self):
         return self.input_params + [param for param in self.output_params
                                     if param not in self.input_params]
 
+    def clean(self, clean_params=False):
+        """Cleans up all the data object parameters in the operator signature
```

Comment:
Slightly misleading docstring, since the primary functionality is to clear the sympy cache.